javatests/arcs/core/host/TestConstructedParticle.kt

```diff
@@ -0,0 +1,23 @@
+package arcs.core.host
+
+import arcs.core.data.Plan
+import arcs.core.data.Schema
+import arcs.core.entity.EntityBaseSpec
+import arcs.sdk.BaseParticle
+import arcs.sdk.HandleHolderBase
+
+fun build(spec: Plan.Particle): TestConstructedParticle {
+    val schema = spec.handles["data"]!!.type.toSchema()
+    return TestConstructedParticle(schema)
+}
+
+class TestConstructedParticle(val schema: Schema) : BaseParticle() {
```

Comment:
Is this particle ever run in the test? If so, can we have some assertion that it actually receives the type properly, e.g. you can make an assertion that the set of fields it sees is what you expect.

Reply:
I added this test recently (today and yesterday).